tests/unit-core/test-snapshot.c

```diff
@@ -195,79 +195,13 @@ static void test_exec_snapshot (uint32_t *snapshot_p, size_t snapshot_size, uint
 int
 main (void)
 {
-  static uint32_t snapshot_buffer[SNAPSHOT_BUFFER_SIZE];
-
   TEST_INIT ();
 
-  /* Dump / execute snapshot */
```

Comment:
What about adding an assertion to the current value of CBC_END, CBC_EXT_END, and a comment before that if this change, the snapshot version should be increased.

Reply:
I resolved this problem via adding static asserts for them.